packages/webpack-config/web-default/index.html

```diff
@@ -19,15 +19,18 @@
         -webkit-overflow-scrolling: touch;
         margin: 0px;
         padding: 0px;
-        /* Allows content to fill the viewport and go beyond the bottom */
-        min-height: 100%;
+        /* DO NOT Allows content to fill the viewport and go beyond the bottom */
```

Comment:
Fix grammer

Reply:
@EvanBacon My english is bad. Please do it your-self.